go/service/account.go

```diff
@@ -260,3 +260,50 @@ func (h *AccountHandler) TimeTravelReset(ctx context.Context, arg keybase1.TimeT
 
 	return err
 }
+
+/////////////////////////
+type GetContactSettingsResponse struct {
+	libkb.AppStatusEmbed                                /// ?
+	Version              int                            `json:"version"`
```

Comment:
This won't match the API result, there is one more level of embedding:

```json
{
  "status": {
    "code": 0,
    "name": "OK"
  },
  "settings": {
    "version": 1,
    "allow_followee_degrees": 1,
    "enabled": true,
    "teams": [
      {
        "team_id": "fa244938df6316634817c4dd0ec09e24",
        "allow_followees_of_team_members": 0,
        "enabled": true
      }
    ]
  }
}
```

As such, the response struct should be:

```go
type GetContactSettingsResponse struct {
	libkb.AppStatusEmbed
    Settings keybase1.ContatSettings `json:"settings"`
}
```